iocage_lib/ioc_common.py

```diff
@@ -690,12 +690,15 @@ def parse_latest_release():
         # We want a dynamic supported
         try:
             if "releng/" in rel[1]:
-                rel = rel[1].strip('</td').strip("releng/")
+                rel = rel[1].strip('</td').strip('</p').strip("releng/")
+                float(rel)
```

Comment:
Is this debug? Why the cast?

Reply:
No debug, this is a catch-all to filter out some potential invalid values.

As an example, there's a line `The Security Branch tags have names like releng/13.0.` in code tags that does not directly to the [Supported FreeBSD releases](https://www.freebsd.org/security/#sup) table that is caught by this.

Reply:
Got it, good stuff